src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { AngularFireModule } from "angularfire2";
+
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
```

Comment:
As I know, `FormsModule` and `ReactiveFormsModule` are independent modules different types of forms (template-driven form or reactive form). do you use both of them?